index.js

```diff
@@ -25,7 +36,14 @@ function fastifySwagger (fastify, opts, next) {
   fastify.decorate('swaggerCSP', require('./static/csp.json'))
 }
 
-module.exports = fp(fastifySwagger, {
+const plugin = fp(fastifySwagger, {
   fastify: '>=3.x',
   name: 'fastify-swagger'
 })
+
+module.exports.default = plugin
+
+module.exports = {
+  fastifySwagger: plugin,
+  validatorCompiler
+}
```

Comment:
please restore the module.export to be the plugin. You fan attach the other properties to
module.exports.

Could you also restore the require of all the tests?

Reply:
we have a default export for plugins if user don't want to have object

Reply:
have a look on line no 44 (index.js)

Reply:
ok, fixed